eng/pipelines/runtime-staging.yml

```diff
@@ -176,6 +176,34 @@ jobs:
         creator: dotnet-bot
         testRunNamePrefixSuffix: Mono_$(_BuildConfig)
 
+#
+# Build Mono and Libraries for Android using Android native crypto instead of OpenSSL
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: Release
+    runtimeFlavor: mono
+    platforms:
+    - Android_x64
+    - Android_x86
+    - Android_arm64
+    variables:
+      # disable using OpenSSL
+      - name: ANDROID_OPENSSL_AAR
+        value: ''
+      # map dependencies variables to local variables
+      - name: librariesContainsChange
+        value: $[ dependencies.evaluate_paths.outputs['SetPathVars_libraries.containsChange'] ]
+    jobParameters:
+      nameSuffix: Libraries_Mono_AndroidCrypto
+      buildArgs: -s mono+libs+libs.tests -c $(_BuildConfig)
```

Comment:
why do we need to build the libraries tests?

Reply:
Ah, I was going to try to run some of the tests originally - but we are definitely not at that point yet. Will remove.

Reply:
If the end goal is to run them, you can leave it if we are going to run them in the near future so that you don't waste cycles on removing it and adding it back later on.